README.md

```diff
@@ -1,16 +1,30 @@
-## Universal Wallet Interoperability Spec
+## Universal Wallet Interoperability
 
 ![CI](https://github.com/w3c-ccg/universal-wallet-interop-spec/workflows/CI/badge.svg)
 
 A portable, extensible, JSON-LD wallet supporting digital currencies and credentials.
 
 ## Links
 
-### [Latest Specification](https://w3c-ccg.github.io/universal-wallet-interop-spec/)
+### Latest Specification
```

Comment:
Should the link to the spec not now be below the section header, instead of zero section content?

Reply:
The link is preserved, see below (the bullet point that makes use of the perma-id redirect service).